python/lsst/pipe/tasks/processCcdWithFakes.py

```diff
@@ -713,7 +753,7 @@ def run(self, fakeCats, exposure, skyMap, wcs=None, photoCalib=None, exposureIdI
                                        ccdVisitFakeMagnitudes=ccdVisitMagnitudes)
         return resultStruct
 
-    def addVariablity(self, fakeCat, band, exposure, photoCalib, exposureIdInfo):
+    def addVariability(self, fakeCat, band, exposure, photoCalib, rngSeed):
```

Comment:
The method name change counts as an API change, but I'm definitely not going to ask you to RFC it or add deprecations!

Reply:
Yeah, I figured the odds were in my favor that nothing else was calling this.

This whole ticket really reminded me of the importance of making your methods private by default and public only when needed, which is something I don't think we've directly encouraged enough and definitely weren't doing when most of pipe_base was written.

Reply:
Yeah, we absolutely should do that more.